internal/events/retrieve.go

```diff
@@ -53,47 +55,47 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			var events []packngo.Event
+			var events []metal.Event
 			var err error
 			header := []string{"ID", "Body", "Type", "Created"}
 			inc := []string{}
+			exc := []string{}
 
 			// only fetch extra details when rendered
 			switch c.Servicer.Format() {
 			case outputs.FormatJSON, outputs.FormatYAML:
 				inc = append(inc, "relationship")
 			}
 
-			listOpt := c.Servicer.ListOptions(inc, nil)
```

Comment:
The `packngo` code called `c.Servicer.ListOptions(inc, nil)`, which would populate the includes and excludes based on the `--include` and `--exclude` command-line flags.

It looks like that was lost in the `metal-go` conversion. @vasubabu could you open a follow-up PR to hook this subcommand up to the `c.Servicer.Include()` and `c.Servicer.Exclude()` functions that you added?